src/main/java/com/vistatec/ocelot/tm/gui/match/AlternateRowsColorRenderer.java

```diff
@@ -0,0 +1,46 @@
+package com.vistatec.ocelot.tm.gui.match;
+
+import java.awt.Color;
+import java.awt.Component;
+
+import javax.swing.JTable;
+import javax.swing.table.DefaultTableCellRenderer;
+
+/**
+ * Table Cell Renderer class, displaying table with alternate rows color.
+ */
+public class AlternateRowsColorRenderer extends DefaultTableCellRenderer {
+
```

Comment:
I'm not sure if this is intentional, but on the Mac this renderer only applies to text cells, not numeric ones, and as a result it looks a bit odd:

![screen shot 2015-05-27 at 3 33 11 pm](https://cloud.githubusercontent.com/assets/641077/7848996/b43ae042-0485-11e5-8d6f-badfcee1bb2c.png)